purview/utils/general.py

```diff
@@ -10,9 +10,12 @@
 
 def timestamp_to_datetime(timestamp):
     """
-    Parses a timestamp and return a datetime object
-    :param timestamp: a string of a timestamp
-    :return: a datetime object
+    Convert a string timestamp to a datetime object.
+
+    :param str timestamp: a timestamp
```

Comment:
"string in timestamp format"?

Reply:
Changing to `:param str timestamp: a generic or ISO-8601 timestamp`